### PR TITLE
fix(estree): rename FieldDefinition -> PropertyDefinition, PrivateName -> PrivateIdentifier

### PR DESCRIPTION
**Pull request by 3cp**
Not sure do we count this as a breaking change for a major release?

### CODE REVIEW

File: src/estree.ts [hunk 1]
```diff
@@ -356,21 +356,23 @@ export interface CatchClause extends _Node {
 
 export interface ClassBody extends _Node {
   type: 'ClassBody';
-  body: (ClassElement | FieldDefinition)[];
+  body: (ClassElement | PropertyDefinition)[];
```
Comment (KFlash) [inline src/estree.ts:359]: This is okay, but I was wondering why the specs are using `PropertyDefinition`. This is part of object literal if you look at the ecma specs. And if you look at the specs for the fields `FieldDefinition` is mentioned there.
The specs mxing two things that isn't related

**Review by KFlash — APPROVED** [review]

File: src/estree.ts (hunk 1 above)
Comment (3cp) [reply src/estree.ts:359]: What do you mean? There is no `FieldDefinition` in current estree spec.

Comment (KFlash) [reply src/estree.ts:359]: That is what I find confusing. It's in the field def .
`PropertyDefinition` has nothing to do with fields. Fields is a Class feature, right?

See the current ECMA specs. It's actually part of ObjectLiteral

https://tc39.es/ecma262/#prod-ObjectLiteral

Comment (3cp) [reply src/estree.ts:359]: That's confusing.

Comment (KFlash) [reply src/estree.ts:359]: Maybe you can drop a comment regarding this in the ESTree repo? My head is elsewhere atm. Working on TS parsing. Maybe something to consider for Meriyah to support?

**Comment by KFlash** [comment]
I guess this is a BC, but do you merge it and publish a new release?

**Comment by 3cp** [comment]
Agree. But in urgent need to release.

**Pull request merged**